admin_guide/quota.adoc

```diff
@@ -476,3 +476,53 @@ If a quota has been defined for your project, see
 xref:../dev_guide/deployments/basic_deployment_operations.adoc#deployment-resources[Deployment Resources]
 for considerations on any deployment configurations.
 endif::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+[[limited-resources-quota]]
+== Require explicit quota to consume a resource
+
+[NOTE]
+====
+This feature is tech preview and subject to change in future releases.
+====
+
+If a resource is not tracked by quota, a user has no restriction
```

Comment:
@derekwaynecarr Can we change "tracked" to "managed"? Other sections of the topic refer to managing a quota.